eng/config/PublishData.json

```
@@ -135,7 +135,18 @@
       ],
       "vsBranch": "main",
       "vsMajorVersion": 17,
-      "insertionTitlePrefix": "[d17.3p2]"
+      "insertionTitlePrefix": "[Validation]",
+      "insertionCreateDraftPR": true
+    },
+    "main-vs-deps": {
+      "nugetKind": [
+        "Shipping",
+        "NonShipping"
+      ],
+      "vsBranch": "main",
+      "vsMajorVersion": 17,
+      "insertionTitlePrefix": "[d17.3p2]",
+      "packageFeeds": "default"
```

It isn't necessary to specify `packageFeeds` when it is `default`
```suggestion
      "insertionTitlePrefix": "[d17.3p2]"
```